internal/grpcutil/target_test.go

```diff
@@ -84,6 +84,14 @@ func TestParseTargetString(t *testing.T) {
 		{targetStr: "unix:/a/b/c", want: resolver.Target{Scheme: "unix", Authority: "", Endpoint: "/a/b/c"}, wantWithDialer: resolver.Target{Scheme: "", Authority: "", Endpoint: "unix:/a/b/c"}},
 		{targetStr: "unix:///a/b/c", want: resolver.Target{Scheme: "unix", Authority: "", Endpoint: "/a/b/c"}},
 
+		{targetStr: "unix-abstract:a/b/c", want: resolver.Target{Scheme: "unix-abstract", Authority: "", Endpoint: "a/b/c"}},
```

Comment:
`unix-abstract:///a/b/c` and `unix-abstract://authority/a/b/c` should parse as expected as well.  I don't believe they will with the changes in this PR currently.

Reply:
Test cases added